src/server/models/MessageModel.ts

```diff
@@ -59,7 +59,7 @@ const messageSchema = new Schema<MessageSchemaDocument>(
       default: false,
     },
     replyTo: {
-      type: Schema.Types.ObjectId,
+      type: String,
```

Comment:
it's not a string, because it's a ref

Reply:
Initially, the replyTo it's an empty unless a value is provided. If the type is Schema.Types.ObjectId, the server throws a BSONerror.  `throw new BSONError('Argument passed in must be a string of 12 bytes or a string of 24 hex characters or an integer');`

Reply:
got it